src/box/box.cc

```diff
@@ -3856,6 +3893,34 @@ bootstrap_journal_write(struct journal *base, struct journal_entry *entry)
 	return 0;
 }
 
+/**
+ * Wait until every remote peer that managed to connect chooses this node as its
+ * bootstrap leader and fail otherwise.
+ */
+static void
+check_bootstrap_unanimosity(void)
```

Comment:
Can't find the word `unanimosity` in a dictionary. What is that? Did you mean `unanimity`?

Reply:
Yep. Google translate gave me `unanimosity`. But I see that `unanimity` is more common. So changed.